tests/src/kakarot/test_state.cairo

```diff
@@ -121,3 +121,44 @@ func test__copy__should_return_new_state_with_same_attributes{
 
     return ();
 }
+
+@external
+func test_is_account_alive_existing_account{
```

Comment:
there is a convention that the tested function is surrouned by double `_`, ie that `__is_account__`